### PR TITLE
feat: [CDS-73716]: add static schema for Aws CDK deploy step

### PR DESCRIPTION
**Pull request by tmacari-harness**
feat: [CDS-73716]: add static schema for Aws CDK deploy step

[CDS-73716]: https://harness.atlassian.net/browse/CDS-73716?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: v0/pipeline/steps/cd/aws-cdk-deploy-step-node.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+title: AwsCdkDeployStepNode
```
Comment (prashantsharma04) [inline v0/pipeline/steps/cd/aws-cdk-deploy-step-node.yaml:1]: move this file to common module steps/common as this is being referred by multiple modules

File: v0/pipeline/steps/cd/aws-cdk-deploy-step-info.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
+title: AwsCdkDeployStepInfo
```
Comment (prashantsharma04) [inline v0/pipeline/steps/cd/aws-cdk-deploy-step-info.yaml:1]: move this file to common module steps/common as this is being referred by multiple modules

Comment (tmacari-harness) [reply v0/pipeline/steps/cd/aws-cdk-deploy-step-info.yaml:1]: should we do that for other  steps also?
If we would look at v0/pipeline/stages/cf/execution-wrapper-config.yaml there are a lot of stages from cd and custom

Comment (tmacari-harness) [reply v0/pipeline/steps/cd/aws-cdk-deploy-step-info.yaml]: Done

File: v0/pipeline/steps/cd/aws-cdk-deploy-step-node.yaml (hunk 1 above)
Comment (tmacari-harness) [reply v0/pipeline/steps/cd/aws-cdk-deploy-step-node.yaml]: Done

**Review by prashantsharma04 — APPROVED** [review]

**Pull request merged**